interpolator/interpolator.F90

```diff
@@ -445,14 +445,25 @@ subroutine interpolator_init( clim_type, file_name, lonb_mod, latb_mod, &
   read (input_nml_file, nml=interpolator_nml, iostat=io)
   ierr = check_nml_error(io,'interpolator_nml')
 
+  ! retain_cm3_bug is no longer supported.
+  if (retain_cm3_bug) then
+    call mpp_error(FATAL, "interpolator_init: You have overridden the default value of retain_cm3_bug " // &
+                          "and set it to .true. in interpolator_nml. This was a temporary workaround " // &
+                          "that is no longer supported. Please remove this namelist variable.")
+  else if (.not. retain_cm3_bug) then
+    call mpp_error(FATAL, "interpolator_init: You have set the outdated namelist variable retain_cm3_bug " // &
```

Comment:
This should not be a FATAL. 

Also `retain_cm3_bug` is initialized to `.true.` so if the variable is not in the namelist the variable is going to be false and you always get this messgae. The same comment applies to all the other namelists that were removed.